src/config.cpp

```diff
@@ -695,22 +695,52 @@ CParseResult CConfig::parse() {
 CParseResult CConfig::parseRawStream(const std::string& stream) {
     CParseResult      result;
 
-    std::string       line    = "";
-    int               linenum = 1;
+    std::string rawLine    = "";
+    int         rawLineNum = 0;
+    std::string line       = "";
+    int         lineNum    = 0;
 
     std::stringstream str(stream);
 
-    while (std::getline(str, line)) {
+    while (std::getline(str, rawLine)) {
```

Comment:
instead of getline into rawLine and then copying the string into line, why not just getline to line? :)